tuned/daemon/daemon.py

```diff
@@ -218,6 +218,7 @@ def _thread_code(self):
 						self._unit_manager.update_monitors()
 						log.debug("performing tunings")
 						self._unit_manager.update_tuning()
+				exports.period_check()
```

Comment:
I think there is still a room for improvement. You could replace the `self._cmd.wait()` above by `epoll` with timeout and sync the time with timer. I.e. if the epoll exits after 1s, then it will be the same. If the epoll exits before 1s, then process the sockets. When the sockets are processed check the timer and run epoll with the lower timeout set.